test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -43,13 +43,12 @@ brokerStart CB 101,38 IPv4 -statTiming
 # 00. Pre-call to GET /statistics to give VALGRIND a possibility
 #
 # 01. Second call to GET /statistics (gives time stat of FIRST request for /statistics)
-# 02. GET /statistics
-# 03. Create entity E2/A1-A5 (identical) with v1 JSON
-# 04. GET /statistics
-# 05. Create entity E3/A1-A5 (identical) with v2 JSON
-# 06. GET /statistics
-# 07. GET /v1/contextTypes
-# 08. GET /statistics
+# 02. Create entity E2/A1-A5 with v1 JSON
+# 03. GET /statistics
+# 04. Create entity E3/A1-A5 (identical) with v2 JSON
+# 05. GET /statistics
+# 06. GET /v1/contextTypes
+# 07. GET /statistics
 #
```

Comment:
Test "script" description has been changed, but no actual changes in the steps... was this due to description was wrong so you have take the opportunity to fix in this PR?